samples/nrf5340/empty_app_core/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  name: Empty firmware for application core
+  description: Minimal Zephyr configuration sample that only suspends CPU.
+tests:
+  test_build:
+    build_only: true
+    build_on_all: true
+    platform_whitelist: nrf5340_dk_nrf5340_cpuapp
```

Comment:
Does this need the _ns board, here too?

Reply:
In my opinion the `_ns` target is not very practical. Network core is started in secure part, so `_ns` target also have no ability to do it. It is also not possible to minimize current consumption by disabling app RAM. It is also not possible to give control over GPIOs to the network core.

Supporting `_ns` target requires `#ifdefs` that will actually cut off everything and this sample become exactly the same as `zephyr/samples/basic/minimal`.

Reply:
OK fair point.